README.md

```diff
@@ -5,3 +5,17 @@ This repository contains the definition of the conda environment used by the sci
 
 It has been moved out from lsstsw package in December 2018 (git clone --bare and git push --mirror).
 
+Procedure
+---------
+
+In order to regerate the pinned files from the bleed ones, following procedure has to be followed:
+
+1. ensure your conda is up to date to version *4.6.8*
+1. make a copy of the bleed txt file to a yaml file: `copy conda3_bleed-osx-64.txt conda3_bleed-osx-64.yml`
+1. edit the yml file and add at the beggining: `name: bleed`
```

Comment:
Why can't we add the name to the `.txt` file? We are only running bleed deployment with conda systems that will understand it.  Similarly, why can't we rename this file to .yml and commit a soft link to .txt for a short time for backwards compatibility (again, bleed deployment is only run by us).

Reply:
I was not sure that the name definition in the txt file will confuse conda when creating the environment, so I think this can be done.